HealthAngels.SignedSessions/Session/SignedDistributedCache.cs

```diff
@@ -8,12 +8,14 @@
 
 namespace HealthAngels.SignedSessions.Session
 {
-    public class CustomRedisCache : IDistributedCache
+    public class SignedDistributedCache : ISignedDistributedCache
     {
         private readonly IDistributedCache _cache;
         private readonly ISignatureHelper _signatureHelper;
 
-        public CustomRedisCache(IDistributedCache cache, ISignatureHelper signatureHelper)
+        public SignedDistributedCache()
```

Comment:
Think this constructor can be removed again (as you have an interface to mock now)